templates/nginx-omero.conf.j2

```diff
@@ -39,9 +39,9 @@ server {
     location / {
         error_page 502 @maintenance;
         # return 502;
-        # checks for static file, if not found proxy to {{ nginx_omero_web_server_address }}
+        # checks for static file, if not found proxy to nginx_omero_web_server_address
     
-        proxy_pass              {{ nginx_omero_web_server_address }};
+        proxy_pass              {{ nginx_omero_web_server_address | default('https://www.openmicroscopy.org/') }};
```

Comment:
This doesn't seem appropriate.

Reply:
I need a working URL, otherwise Nginx won't start in the molecule tests